docs/DevelopmentSetupAndroid.md

```diff
@@ -78,3 +78,22 @@ Genymotion is much easier to set up than stock Google emulators. However, it's o
   ![Create AVD dialog](/react-native/img/CreateAVD.png)
   2. With the new AVD selected, click `Start...`
 5. To bring up the developer menu press F2
+
+### Windows Hyper-V Alternative
+
+The [Visual Studio Emulator for Android](https://www.visualstudio.com/en-us/features/msft-android-emulator-vs.aspx) is a free android emulator that is hardware accelerated via Hyper-V. It doesn't require you to install Visual Studio at all.
+
+It is free, very powerful, fast and hardware accelerated via Hyper-V technology.
```

Comment:
Okay, missed this one also. We basically say the same things in the previous paragraph. So remove this also.